src/test/java/gov/nasa/cumulus/metadata/test/MetadataFilesToEchoTest.java

```diff
@@ -370,4 +370,40 @@ public void testReadIsoMendsMetadataFile() {
         }
     }
 
+    @Test
+    public void testCreateJsonWithPNGFile()
+            throws ParseException, IOException, URISyntaxException {
+        ClassLoader classLoader = getClass().getClassLoader();
+        File file = new File(classLoader.getResource("MODIS_T-JPL-L2P-v2014.0.cmr.cfg").getFile());
+        MetadataFilesToEcho mfte = new MetadataFilesToEcho();
+        try {
+            mfte.readConfiguration(file.getAbsolutePath());
+        } catch (Exception e) {
+            e.printStackTrace();
```

Comment:
Rather than try/catching these calls, would it make more sense to let the exception get raised and the test fail?

Reply:
The test does fail though (line 383) with `fail();`

Reply:
Right, so why catch the exception only to then `fail()`? The test will fail automatically if an exception is thrown, so this is redundant. I also believe the exception stack trace will be already be printed to the screen without `e.printStackTrace` since we're in a unit test

Reply:
Ah gotcha; yeah I believe it was left over when first writing the test and used for debugging purposes; removed the whole try/catch blocks

Reply:
Thank you @hkryeung, that looks much cleaner. Appreciate it!